README.md

```diff
@@ -113,6 +114,25 @@ Command for generate an Action inside your Domain directory.
 
 ---
 
+#### `domain:make:builder UserBuilder User`
+Command for generate a Query Builder inside your Domain directory.
+
+**Arguments**
+
+|  Name  |    Description     |
+|:------:|:------------------:|
+|  Name  | Builder class name |
+| Domain |    Domain Name     |
+
+**Options**
+
+|       Name        |             Description              |
+|:-----------------:|:------------------------------------:|
+| --model=ModelName | To hint Model class on Query Builder |
+|      --force      |    Force create the Builder class    |
+
+---
+
```

Comment:
probably you can add the docs how to make the builder work in model

Reply:
Oh okay, I'll add it